whois-nrtm4/src/main/java/net/ripe/db/nrtm4/Nrtmv4Condition.java

```diff
@@ -0,0 +1,16 @@
+package net.ripe.db.nrtm4;
+
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+
+public class Nrtmv4Condition implements Condition {
+
+    @Override
+    public boolean matches(ConditionContext context, AnnotatedTypeMetadata metadata) {
+        if( !context.getEnvironment().containsProperty("nrtm4.enabled")) {
```

Comment:
if the property nrtm4.enabled is false and therefore we do not complies the condition what is going to happen?

Reply:
the bean is not going to be generated so the job is not going to run